src/module/actor/data-model-character.test.js

```diff
@@ -1,60 +1,47 @@
-import OseDataModelCharacter from "./data-model-character"
+/**
+ * @file Quench unit tests for the Character data model class.
+ */
+export const key = "ose.datamodel.character";
+export const options = { displayName: "Character Data Model" };
 
-export const key = 'ose.datamodel.character';
-export const options = { displayName: 'Character Data Model'}
+// @todo How can we add items to this actor?
+const createMockCharacter = (system = {}, items = [], characterOptions = {}) =>
+  // eslint-disable-next-line new-cap
+  new Actor.implementation({
+    name: `Mock Character ${foundry.utils.randomID()}`,
+    type: "character",
+    ...characterOptions,
+    system,
+    items,
+  });
 
-export default ({
-  before,
-  beforeEach,
-  after,
-  describe,
-  it,
-  expect,
-  ...context
-}) => {
-	// @todo How can we add items to this actor?
-	const createMockCharacter = (system = {}, items = [], options = {}) =>	
-		new Actor.implementation({
-			name: `Mock Character ${foundry.utils.randomID()}`,
-			type: 'character',
-			...options,
-			system,
-			items
-		})
-	
-  const createMockItem = (type, weight, quantity, options = {}) => 
```

Comment:
createMockItem is gone or moved?

Reply:
It's been removed, I don't think we were using it.